pulsar-common/src/main/java/org/apache/pulsar/policies/data/loadbalancer/LocalBrokerData.java

```diff
@@ -249,14 +249,21 @@ public String printResourceUsage() {
                 cpu.percentUsage(), memory.percentUsage(), directMemory.percentUsage(), bandwidthIn.percentUsage(),
                 bandwidthOut.percentUsage());
     }
-
+    @Deprecated
```

Comment:
Nit: what do you think about adding the `since = "3.0.0"` arg to the `@Deprecated` annotation. I did that in a recent PR because it has the benefit of making it clearer when we can remove it later. The history is clear in git, but this lets a user skip a few steps.

Reply:
Sorry, I see now that the `since` parameter was introduced in Java 9, and we require `pulsar-common` has Java 8 compatibility. Thanks for trying it out!